docs/concepts/services-networking/service.md

```diff
@@ -708,6 +700,10 @@ work, and the client IP is not altered.
 This same basic flow executes when traffic comes in through a node-port or
 through a load-balancer, though in those cases the client IP does get altered.
 
+#### Ipvs
+
+The same as Iptables mostly, the most different thing is that  ipvs use hash table as the underlying data structure, making less routing rules and faster than iptables.
```

Comment:
I would say,

"Iptables operations slow down dramatically in large scale cluster e.g 10,000 Services.  IPVS is designed for load balancing and based on in-kernel hash tables. So we can achieve performance consistency in large number of services from IPVS-based kube-proxy. Meanwhile, IPVS-based kube-proxy has more sophisticated load balancing algorithms (least conns, locality, weighted, persistence)

Reply:
Done